txscript/opcode.go

```diff
@@ -1009,6 +1010,30 @@ func opcodeReturn(op *parsedOpcode, vm *Engine) error {
 	return ErrStackEarlyReturn
 }
 
+// verifyLockTime is a helper function used to validate locktimes.
+func verifyLockTime(txLockTime, threshold, lockTime int64) error {
+	// The lock time field of a transaction is either a block height at
```

Comment:
This first part of this comment is no longer accurate since this function is used for both `opcodeCheckLockTimeVerify` and `opcodeCheckSequenceVerify`.  Please move it to the call site in `opcodeCheckLockTimeVerify`.

That will only leave this section here:
```Go
	// The lockTimes in both the script and transaction must be of the same
	// type.
```